app/views/layouts/home/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Hello BrightCoders</h1>
```

Comment:
Please check my last comment about adding a new line at the of the files and let me know if you agree.